### PR TITLE
ci: merge stream-chat-css docs into React docs

### LINKED ISSUE #1577: Set up infrastructure for theming-v2
**Motivation**
A clear and concise description of what the problem/opportunity is. Ex. I'm always frustrated when [...]

**Proposed solution**
- Create a working branch for theming-v2 (to not block `develop`)
- Encapsulate theme-v1 and theme-v2 -> integrators can choose which version to import (Angular has this setup: https://github.com/GetStream/stream-chat-angular/blob/theming-v2/package.json#L37)
- Create initial documentation for theming ([we might be able to use a shared documentation](https://github.com/GetStream/stream-chat-css/issues/99), but right now we use the suboptimal solution of copy-pasting from Angular), and deprecate the old documentation - https://github.com/GetStream/stream-chat-angular/tree/document-theme-variables/docusaurus/docs/Angular/theming 

**Acceptance Criteria**
A set of predefined requirements that must be met to mark a user story complete.

### PR DESCRIPTION
**Pull request by MartinCupela**
### 🎯 Goal

Merge contents of the `stream-chat-css/docs/` contents into `stream-chat-react/docusaurus/docs/React` folder 

1. while running CI (`.github/workflows/docusaurus.yml`)
2. for local development (`package.json` script `docs-merge`)

fixes #1577

### CODE REVIEW

File: scripts/merge-stream-chat-css-docs.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+main() {
+  if [ $# -eq 0 ]; then
+      usage
+      exit 0
+  fi
+
+  STREAM_CHAT_CSS_PATH=$1;
+  cp -r "$STREAM_CHAT_CSS_PATH"/* ./docusaurus/docs/React;
```
Comment (szuperaz) [inline scripts/merge-stream-chat-css-docs.sh:15]: Will this merge the assets folder from the React SDK folder and the CSS folder?

Comment (MartinCupela) [reply scripts/merge-stream-chat-css-docs.sh:15]: @szuperaz yes, but for clarity I will rename the var to STREAM_CHAT_CSS_DOCS_PATH.

Comment (szuperaz) [reply scripts/merge-stream-chat-css-docs.sh:15]: I was asking if it's a problem that both folders had folders named `assets`, but I based on google the `cp` command can handle that

**Review by szuperaz — APPROVED** [review]

**Review by arnautov-anton — APPROVED** [review]

**Pull request merged**